### PR TITLE
[Content migration] - Asset Conversion

### PR DESCRIPTION
**Pull request by nhussein11**
This PR aims to bring content from the `polkadot-ecosystem-docs-draft repository`

Migrated content:

- Asset Conversion on Asset Hub

According to the spreadsheet, the location of this guide should be:
`Polkadot Protocol > System Parachains > Asset Hub > Asset Conversion`

### CODE REVIEW

**Comment by nhussein11** [comment]
@eshaben I'm unsure anyways, since this content from my point of view is more "practical" and it's falling under the `Polkadot Protocol` section

**Comment by eshaben** [comment]
Yeah, let's not worry about that now. We can move it later 🙂

File: polkadot-protocol/system-parachains/asset-hub/asset-conversion.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+         - **`0: StagingXcmV3MultiLocation`** - the Multilocation of the first asset in the pool. In this case, it is the PPM token, which the following Multilocation represents:
+        ```javascript
+        {
+            parents: 0,
+            interior: {
+                X2: [{PalletInstance: 50}, {GeneralIndex: 1112}]
+            }
+        }
+        ```
+        - **1: StagingXcmV3MultiLocation** - the second asset's Multilocation within the pool. This refers to the DOT token, which the following Multilocation identifies:
```
Comment (eshaben) [inline polkadot-protocol/system-parachains/asset-hub/asset-conversion.md:231]:
```suggestion
        - **`1: StagingXcmV3MultiLocation`** - the second asset's Multilocation within the pool. This refers to the DOT token, which the following Multilocation identifies: 
```

**Review by eshaben — COMMENTED** [review]
Did a quick scan on this one since it was previously reviewed in the other repo. Just noticed one minor thing

**Review by eshaben — APPROVED** [review]
LGTM!

**Pull request merged**